package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "nodejs_epam",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DzmitryBiruk/nodejs_epam.git"
+  },
+  "author": "Dzmitry Biruk",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/DzmitryBiruk/nodejs_epam/issues"
+  },
+  "homepage": "https://github.com/DzmitryBiruk/nodejs_epam#readme",
+  "babel": {
```

Comment:
Export into ‘.babelrc’, please. Good practice is ‘one file - one responsibility’.